src/js/views/panel_view.js

```diff
@@ -357,6 +365,56 @@
             this.render_layout();
         },
 
+        render_colorbar: async function() {
+
+            if (this.$colorbar) {
+                this.$colorbar.remove();
+            }
+            var cb = this.model.get('colorbar');
+            var start = 0,
+                end = 125,
+                reverseIntensity = false,
+                color = "";
+            for (const chann of this.model.get('channels')) {
+                if(chann.active) {
+                    color = chann.color;
+                    start = chann.window?.start;
+                    end = chann.window?.end;
+                    reverseIntensity = chann.reverseIntensity;
+                    break;
+                }
+            }
+            var lut_url = await FigureLutPicker.loadLuts(false)  // Ensure lut url and list are loaded
```

Comment:
Wow, this looks great. Haven't tested yet, but just noticed you don't need the `false` here (or below) since that's the default.

Reply:
I am not passing the parameter in the two occurences. I keet the parameter in `loadLuts()` in case, but it is not currently in use.